File: code/a2209.pas
program Bai209;
  var a : array[1..1000] of integer;
  i,n: integer;
 begin
  readln(n);
  for i:= 1 to n do read(a[i]);
  for i := 2 to n do 
      if a[i] < a[i - 1] then 
          begin
          writeln('NO');
          exit;
          end
      else  
        begin
          writeln('YES');
          exit;
       end;
end.

File: code/a2209.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,i;
    int a[1000000];
    cin>>n;
    for (i=1; i<=n; i++)
        cin>>a[i];
    for (i=2; i<=n; i++)
        if (a[i]<a[i-1]){
            cout<<"NO";
            return 0;
        }
    cout<<"YES";
    return 0;
}